pages/button/button.css

```diff
@@ -0,0 +1,168 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*, ::before, ::after {
```

Comment:
You can move this piece of code to reset.css where you'll reset all the browser styles
Otherwise, you'll have to repeat this in every component